definitions/contested/json/CaseTypeTab/CaseTypeTab.json

```diff
@@ -2219,6 +2219,15 @@
     "CaseFieldID": "specificArrangementsRequired",
     "TabFieldDisplayOrder": 31
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyContested",
+    "TabID": "hiddenTab",
+    "TabLabel": "Hidden Tab",
```

Comment:
whats going on with the hidden tab stuff? does it need to be named hidden tab and the field the hide is based on so it'll show up as 'Consent Order Process' from here?

`  {
    "LiveFrom": "01/01/2017",
    "CaseTypeID": "FinancialRemedyContested",
    "TabID": "consentOrderProcessTab",
    "TabLabel": "Consent Order Process",
    "TabDisplayOrder": 17,
    "CaseFieldID": "tabConsentOrderProcess",
    "TabFieldDisplayOrder": 1,
    "TabShowCondition": "consentD81Question=\"Yes\" OR consentD81Question=\"No\""
  },`

Reply:
for some reason the field isn't picked up when used in the logic you've pasted unless it's in the hidden tab, I know... dreadful

Reply:
well it's good to know!